net/smcroute/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2019 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=smcroute
+PKG_VERSION:=2.4.4
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-2.0-or-later
```

Comment:
Please include license file

Reply:
Other packages with the same license (I see it is denoted as "GPL-2.0+"), but do not include the license text itself. Can you explain what you expect please?

Reply:
https://github.com/openwrt/packages/blob/master/sound/fdk-aac/Makefile#L13

Reply:
ok, I've added PKG_LICENSE_FILES.